pandas/core/frame.py

```diff
@@ -3501,27 +3501,37 @@ def notnull(self):
     def dropna(self, axis=0, how='any', thresh=None, subset=None,
                inplace=False):
         """
+        Remove missing values.
+
         Return object with labels on given axis omitted where alternately any
-        or all of the data are missing
+        or all of the data are missing.
```

Comment:
It was already there, but I am not fully sure what is meant with "alternately" ? (that implies something alternating)

Reply:
Think it should mean "alternative". But will change it to "either", because should be easier to understand.